README.md

```diff
@@ -14,9 +14,10 @@ The toolchains for the following target architectures are supported:
 - RISC-V (32-bit and 64-bit; RV32I, RV32E, RV64I)
 - SPARC (32-bit and 64-bit; SPARC V8, SPARC V9)
 - x86 (32-bit and 64-bit)
-- Xtensa (sample_controller, intel_ace15_mtpm, intel_tgl_adsp,
-  nxp_imx_adsp, nxp_imx8m_adsp, nxp_imx8ulp_adsp, nxp_rt500_adsp,
-  espressif_esp32, espressif_esp32s2, espressif_esp32s3, mt8195_adsp)
+- Xtensa (sample_controller, amd_acp_6_0_adsp, intel_ace15_mtpm,
+  intel_tgl_adsp, nxp_imx_adsp, nxp_imx8m_adsp, nxp_imx8ulp_adsp,
+  nxp_rt500_adsp, espressif_esp32, espressif_esp32s2, espressif_esp32s3,
+  mt8195_adsp)
```

Comment:
[A recent update](https://github.com/zephyrproject-rtos/sdk-ng/commit/fc0f246304007de374fe1d40f6682e950f2369df) removed the full enumeration of Xtensa toolchain variants.

This hunk can be removed.

Reply:
Does the topic-amd_acp_6_0 branch need to be updated before I apply my changes?

Reply:
You can push directly to the upstream `topic-amd_acp_6_0`, or alternatively open a PR to `main` from your fork branch with the final changes.